bin/http-server

```diff
@@ -75,6 +82,22 @@ var port = argv.p || argv.port || parseInt(process.env.PORT, 10),
     version = argv.v || argv.version,
     logger;
 
+
+if (nopanic){
+
+    process.on('error', (e)=> {
+      let etime = new Date().toISOString().replace(/T/, ' ').replace(/\..+/, '')   
+      console.log(colors.green(etime)) 
+      console.log(colors.red("Fatal error: ")+ e.code + ": "+e.message)
+      let fname = ".httpserver-"+etime.split(" ")[0]+"-"+etime.split(" ")[1]
+      console.log(colors.bold(`Check ${fname} file in this folder.`))
+      fs.writeFileSync(fname, JSON.stringify(e))
+      process.exit(1)
+});
+
+}
+
+
```

Comment:
I have multiple comments on this block, but suffice to say, what do you think of this change:

```suggestion
if (nopanic){
  process.on('error', (e)=> {
    // Results in a string like "2021-12-27 14:56:31"
    const etime = new Date().toISOString().replace(/T/, ' ').replace(/\..+/, '');
    console.log(colors.green(etime));
    console.log(`${colors.red('Fatal error: ')}${e.code}: ${e.message}`);
    const filename = `.httpserver-${etime.split(' ').join('-')}`;
    console.log(colors.bold(`Check ${filename} file in this folder.`));
    fs.writeFileSync(filename, JSON.stringify(e));
    process.exit(1);
  });
}

```